src/measure_theory/probability_mass_function/monad.lean

```diff
@@ -18,8 +18,10 @@ so that the second argument only needs to be defined on the support of the first
 
 -/
 
+universes u v
+
 noncomputable theory
-variables {α β γ : Type*}
+variables {α : Type*} {β : Type*} {γ : Type*}
```

Comment:
Note that this means exactly the same thing as what was written before, so there's no need to change it

Reply:
Just changed this to match the other `pmf` files, probably doesn't matter much I guess. Changed it to switch them all to the simple version